OracleLinux/7/README.md

```diff
@@ -1,36 +1,89 @@
 # ol7-vagrant
-A vagrant box that provisions Oracle Linux automatically, using Vagrant, an Oracle Linux 7 box and a shell script.
+
+A vagrant project that provisions Oracle Linux automatically, using Vagrant, an Oracle Linux 7 box and a shell script.
```

Comment:
This could perhaps be better phrased as "A Vagrant project to automatically build an Oracle Linux 7 virtual machine using either VirtualBox or libvirtd with optional extras including a LAMP stack or container runtime."